service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerTest.java

```diff
@@ -73,7 +75,7 @@
 class DraftOrdersControllerTest extends AbstractControllerTest {
     private static final Long CASE_ID = 1L;
     private static final String SEND_DOCUMENT_EVENT = "internal-change:SEND_DOCUMENT";
-
+    private static final DateTimeFormatter DATE_FORMAT = ofPattern("dd MMMM yyyy, hh:mma");
```

Comment:
I begun doing some work in DateFormatterService, I believe there may be a constant for this already in there that you can static import. If not it is flagged as tech debt I believe so we can address in that

Reply:
ok I will use that